src/components/inputs/LDatePickerDay.vue

```diff
@@ -154,39 +175,45 @@ export default {
       secondBlocked: false,
       rangeLimit: { min: null, max: null },
       temporaryDate: null,
-      datepickerRefsEnum: DATEPICKER_REFS
+      datepickerRefsEnum: DATEPICKER_REFS,
+      typedDate: null
     }
   },
   computed: {
+    dateFormatDictionary () {
+      return  {
+        mask: DATEPICKER_DAY_MASK[this.locale] || DATEPICKER_DAY_MASK.pt,
+        format: DATEPICKER_DAY_FORMAT[this.locale] || DATEPICKER_DAY_FORMAT.pt,
+      }
+    },
     formattedMonths () {
       if (!this.monthsPeriod) {
         return
       }
 
+      const { format } = this.dateFormatDictionary
       const { temporaryDate } = this
       if (temporaryDate && isEmpty(this.monthsPeriod)) {
         return formatYearMonthDay([temporaryDate])
       }
 
       if (temporaryDate && this.monthsPeriod.length === 1 ) {
         const orderedDates = sortDateISO([temporaryDate, ...this.monthsPeriod])
-        return formatYearMonthDay(orderedDates.join(' - '))
+        return formatYearMonthDay(orderedDates.join(' - '), format)
```

Comment:
```suggestion
        const defaultDateSeparator = ' - '
        ...
        return formatYearMonthDay(orderedDates.join(defaultDateSeparator), format)
```
Apenas como sugestão: como o ' - ' é utilizado em vários pontos, talvez poderíamos armazená-lo em uma const local, ou até mesmo em um "SeparatorEnum" para utilizar em outros arquivos :+1: